### ISSUE TITLE
Block Entities do not render with Litematica installed

### ISSUE THREAD

**Issue by CodeF53** [ValkyrienSkies/Valkyrien-Skies-2#540]
### Mod Name

litematica

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.1

### Mod Loader

Fabric

### Issue description

When turning Block entities like Chests/Beds into ships, I expect them to be visible

They aren't with Litematica installed. 

### Issue reproduction

1. install malilib, litematica, Valkyrien Skies, and Fabric API
2. boot game, create world
3. get the "Ship Creator" stick from the "Valkyrien Skies" tab of the creative menu
4. place a chest and shift-right click it with the "Ship Creator"
5. observe the block turning invisible

https://github.com/ValkyrienSkies/Valkyrien-Skies-2/assets/37855219/0ccd4d7d-93d2-415f-b3ca-64947b1fcadf

### Logs

[latest.log](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/files/12065374/latest.log)

The launcher I use didn't seem to make a `debug.log`.

**Comment by StewStrong** [comment]
Can you check if this issue is still occurring in the latest 1.20 version?

**Comment by CodeF53** [comment]
> Can you check if this issue is still occurring in the latest 1.20 version?

Yes it does still happen

```
- fabric-api 0.85.0+1.20.1 
  |-- fabric-api-base 0.4.29+b04edc7a77 
  |-- ...
  \-- fabric-transitive-access-wideners-v1 4.2.0+b3afc78b77 
- fabricloader 0.14.21 
- java 17 
- litematica 0.15.3 
- malilib 0.16.1 
- minecraft 1.20.1 
- valkyrienskies 2.3.0-beta.4 
  |-- cloth-config 11.0.99 
  |    \-- cloth-basic-math 0.6.1 
  |-- com_github_llamalad7_mixinextras 0.1.1 
  \-- fabric-language-kotlin 1.8.5+kotlin.1.7.20 
    |-- org_jetbrains_kotlin_kotlin-reflect 1.7.20 
    |-- ...
    \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.4.1
```

**Comment by thetheaplant** [comment]
Fixed in #819
